packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,33 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
+```sh
+npm install
+```
+### Run
+To start the project is as easy as this
+```sh
+npm run start
+```
+
+### Documentation
+Loopback has a large place at your disposal to document everything you can do that is constantly updating so you never stop trying new things
+> [Loopback Documentation Here](https://loopback.io/doc/en/lb4/)
+
+### Support
```

Comment:
@raymondfeng, there's been some discussion on opening our Slack up. Perhaps it's a good place to advertise?

@frbuceta, to be honest, StackOverflow is _very lightly_ monitored by the team, so I'm a bit reluctant to mention it in the template.

Reply:
Then we put slack as a tool to help our users?

Reply:
sure. 

For community support, open an issue on https://github.com/strongloop/loopback-next. You can also join the [LoopBack Slack workspace](http://loopbackio.slack.com/) via [this invitation link](https://join.slack.com/t/loopbackio/shared_invite/zt-8lbow73r-SKAKz61Vdao~_rGf91pcsw). We also encourage the community helping each other out!

Reply:
Thanks for helping me and for being able to help